.github/workflows/tf_apply_production.yml.disabled

```diff
@@ -0,0 +1,69 @@
+name: "Terraform apply production"
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+    paths:
+      - "terragrunt/env/production/**"
```

Comment:
I would also include the github action name as well in the paths since you want an apply to happen when the workflow is changed.